tests/unit/pages/subjects/Subjects.spec.jsx

```diff
@@ -0,0 +1,16 @@
+import { renderWithProviders } from '~tests/test-utils'
+import Subjects from '~/pages/subjects/Subjects'
+import { screen } from '@testing-library/react'
+
+describe('SubjectPage test', () => {
+  it('It should render page', () => {
+    renderWithProviders(<Subjects />)
+    const pageTitle = screen.getByText('Subjects')
+    expect(pageTitle).toBeInTheDocument()
+  })
+  it('Text of the page must be in the document', () => {
```

Comment:
Why do we need two identical tests?

Reply:
Because task required two separate scenarios for checking if the page renders correctly and if the text of the page is in the document. These two tests look identical because the subjects page hasn't been implemented yet.

Reply:
If you mock `<PageWrapper>` with test id, you can use `getByTest` id fir the first `it` and `getByText` for the second one.